### DISCUSSION TITLE
Live Update: Can Code Signing be used in self-hosted or manual version?

### DISCUSSION THREAD

**Discussion by luisbytes** [capawesome-team/capacitor-plugins#313] [Q&A]
I would like to know if it is possible to sign the zips and validate them when downloading them.

**Comment by robingenz** [comment]
Not yet, but that is also a quick win. Feel free to create a feature request and I will add support for it.

**Reply by robingenz** [reply]
I was wrong. This should already be possible. The download url just needs to return a response header named "X-Signature". This header is also used by Capawesome Cloud.

**Reply by luisbytes** [reply]
It would be interesting to create a guide about it, on the other hand in that case the CLI would also be a little more open, currently to generate the encrypted bundle you must be logged in.

**Reply by robingenz** [reply]
You're right. I will add some documentation. Regarding the CLI: Feel free to create a feature request in https://github.com/capawesome-team/cli. I will then take a look at it.
